lib/Controller/ChattyLLMController.php

```diff
@@ -283,6 +283,7 @@ public function newMessage(int $sessionId, string $role, string $content, int $t
 			$message->setRole($role);
 			$message->setContent($content);
 			$message->setTimestamp($timestamp);
+			$message->setSources('[]');
```

Comment:
it should be automatically be populated by the default value, no?
https://github.com/nextcloud/assistant/blob/e1ab3907439b8158d64218b22edf4dacee961b43/lib/Migration/Version020500Date20250425125359.php#L35

Reply:
Feel free to test. I'm not sure why that doesn't work.

Reply:
the message gets returned without refetching it from the db. In the db the default is applied, but not in the message object

Reply:
ah alright that makes sense. Nice one.